prime-router/metadata/fhir_mapping/hl7/segments/MSH/Provenance-Source.yml

```diff
@@ -0,0 +1,95 @@
+resourceType: Provenance
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+target:
```

Comment:
It looks like this should also point to the created MessageHeader?

Reply:
Good catch, also didn't verify this in the integration tests, but it seems like the bundle reference isn't working either. Will have to investigate and regenerate FHIR again.